pkgs/development/python-modules/myfitnesspal/default.nix

```diff
@@ -11,14 +11,20 @@ buildPythonPackage rec {
     sha256 = "c2275e91c794a3569a76c47c78cf2ff04d7f569a98558227e899ead7b30af0d6";
   };
 
-  # Remove overly restrictive version constraints on keyring and keyrings.alt
+  # Remove overly restrictive version constraints
   postPatch = ''
     sed -i 's/keyring>=.*/keyring/' requirements.txt
+    sed -i 's/rich>=.*/rich/' requirements.txt
     sed -i 's/keyrings.alt>=.*/keyrings.alt/' requirements.txt
   '';
 
-  checkInputs = [ mock nose ];
-  propagatedBuildInputs = [ blessed keyring keyrings-alt lxml measurement python-dateutil requests six ];
+  checkInputs = [ pytest mock nose ];
+  propagatedBuildInputs = [ blessed keyring keyrings-alt lxml measurement python-dateutil requests six rich ];
+
+  # Integration tests require an account to be set
+  preCheck = ''
+    rm tests/test_integration.py
```

Comment:
Can you disable the test with:
```
  disabledTests = [ "test_integration" ];
```

You might need to set ``checkInputs = [ pytestCheckHook ];``

See https://github.com/NixOS/nixpkgs/blob/fa21e8c1bb8b785c042ff0c111d0f684a8d1305b/pkgs/development/python-modules/arpeggio/default.nix#L26

Reply:
Wasn't aware of this, thanks!